xlators/mgmt/glusterd/src/glusterd-op-sm.c

```diff
@@ -2948,12 +2948,14 @@ glusterd_op_set_volume(dict_t *dict, char **errstr)
         }
 
         if (strcmp(key, "config.memory-accounting") == 0) {
-            ret = gf_string2boolean(value, &volinfo->memory_accounting);
+            gf_boolean_t val = _gf_false;
```

Comment:
I think `GF_VOLINFO_MEMORY_ACCOUNTING` could be already set here, so we need to clear it when the boolean is false.

Reply:
OK